osmaxx/conversion/converters/converter_gis/bootstrap/bootstrap.py

```diff
@@ -48,7 +52,7 @@ def bootstrap(self):
 
     @property
     def geom(self):
-        return self._extent
+        return polyfile_helpers.parse_poly_string(self.area_polyfile_string)
```

Comment:
probably not, but it shouldn't be done in `__init__` since we don't always use it.